docs/project-model/project-json-reference.md

```diff
@@ -0,0 +1,494 @@
+    * [allowUnsafe](#allowUnsafe)
+    * [platform](#platform)
+    * [warningsAsErrors](#warningsAsErrors)
+    * [optimize](#optimize)
+    * [keyFile](#keyFile)
+    * [delaySign](#delaySign)
+    * [publicSign](#publicSign)
+    * [emitEntryPoint](#emitEntryPoint)
+    * [xmlDoc](#xmlDoc)
+    * [preserveCompilationContext](#preserveCompilationContext)
+* [analyzerOptions](#analyzerOptions)
+    * [languageId](#languageId)
+* [configurations](#configurations)
+* [frameworks](#frameworks)
+    * [dependencies](#dependencies)
+    * [frameworkAssemblies](#frameworkAssemblies)
+    * [wrappedProject](#wrappedProject)
+    * [imports](#imports)
+    * [bin](#bin)
+        * [assembly](#assembly)
+
+<a name="name"></a>
+## name
+Type: String
+
+The name of the project, this is also the name of the (nuget) package the project will produce.
+
+<a name="version"></a>
+## version
+Type: String
+
+The semver version of the project, also used for the nuget package.
+
+<a name="description"></a>
+## description
+Type: String
+
+A longer decsiption of the project.
+
+<a name="summary"></a>
+## summary
+Type: String
+
+A short description of the project.
+
+<a name="copyright"></a>
+## copyright
+Type: String
+
+The copyright information for the project.
+
+<a name="title"></a>
+## title
+Type: String
+
+The friendly name of the project, can contain spaces and special characters not allowed in the name.
+
+<a name="entryPoint"></a>
+## entryPoint
+Type: String
+
+The entrypoint method for the project, typically Main().
+
+<a name="projectUrl"></a>
+## projectUrl
+Type: String
+
+The url for the homepage of the project.
+
+<a name="licenseUrl"></a>
+## licenseUrl
+Type: String
+
+The url for the licence the project uses.
+
+<a name="iconUrl"></a>
+## iconUrl
+Type: String
+
+The url for an icon that will be used in various places such as the package explorer.
+
+<a name="compilerName"></a>
+## compilerName
+Type: String
+
+The name of the compiler used for this project, for c# this will be "csc" for example.
+
+<a name="testRunner"></a>
+## testRunner
+Type: String
+
+The name of the testrunner to use with this project, setting this also marks the project as a test project.
+
+<a name="authors"></a>
+## authors
+Type: String[]
+
+An array of strings with the names of the authors.
+
+<a name="owners"></a>
+## owners
+Type: String[]
+
+An array of strings with the names of the owners.
+
+<a name="tags"></a>
+## tags
+Type: String[]
+
+An array of strings with tags for the project, used for searching in nuget.
+
+<a name="language"></a>
+## language
+Type: String
+
+The (human) language of the project, corresponds to the "neutral-language" compiler argument.
+
+<a name="releaseNotes"></a>
+## releaseNotes
+Type: String
+
+Release notes for the project.
+
+<a name="requireLicenseAcceptance"></a>
+## requireLicenseAcceptance
+Type: Boolean
+
+A boolean that causes a prompt to accept the package license when installing the package to be shown.
```

Comment:
We'll have to specify that this is *only* relevant to projects that will be NuGet packages.